### PR TITLE
Niloofar / Fix ts errors in academy/blog

### PR DESCRIPTION
**Pull request by niloo-fs**
Changes:

-   ...

## Type of change

-   [x] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/academy/blog/index.tsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ import { AllArticlesQuery } from 'types/graphql.types'
 import Layout from 'components/layout/layout'
 import { SEO, Container, Flex } from 'components/containers'
 import { Header } from 'components/elements'
-import { localize, WithIntl } from 'components/localization'
+import { localize } from 'components/localization'
```
Comment (mohammad-rework) [inline src/pages/academy/blog/index.tsx:11]: For using `localize` function let's check with @swati to see if we need translated `title` and `description` for our SEO tag. If we don't need it we can remove `localize` function from all Academy components.

Comment (niloo-fs) [reply src/pages/academy/blog/index.tsx:11]: Thank u @mohammad-rework for your revision, I think what you said is out of the scope of this card, I just want to fix ts errors here.
do we need to check it here?

Comment (mohammad-rework) [reply src/pages/academy/blog/index.tsx:11]: If we are only focusing on TS errors in here, we can skip it.

**Review by mohammad-rework — APPROVED** [review]

**Review by eduard-deriv — APPROVED** [review]

**Comment by eduard-deriv** [comment]
LGTM 👍

**Pull request merged**